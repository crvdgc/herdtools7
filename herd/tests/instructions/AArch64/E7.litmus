AArch64 E7
(* IRG with all ones, forall *)
Variant=mte
{
int x=100;
0:X0=x;
}
 P0            ;
 MOV W2,#65535 ; (* 0xFFFF *)
 IRG X1,X0,X2  ;
forall 0:X1=x:t0

