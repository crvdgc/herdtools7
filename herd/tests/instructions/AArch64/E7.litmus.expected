Test E7 Required
States 1
0:X1=x:t0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=x:t0)
Observation E7 Always 1 0
Hash=de17bcb7d83fd113de3e5f72449bdb86

